src/Twilio/Http/GuzzleClient.php

```diff
@@ -23,7 +23,7 @@ public function __construct(ClientInterface $client) {
     public function request(string $method, string $url,
                             array $params = [], array $data = [], array $headers = [],
                             string $user = null, string $password = null,
-                            int $timeout = null): Response {
+                            int $timeout = null, string $loglevel = null): Response {
```

Comment:
Why don't we actually do any logging in this client?

Reply:
From my understanding this client already has built in [debugging](https://docs.guzzlephp.org/en/stable/request-options.html#debug). I had to include loglevel as a parameter in this client's request to match what the client.php is expecting.